### PR TITLE
Package upgrades

### PR DESCRIPTION
**Pull request by jasongrout**
Upgrade

* typescript and tslint
* phosphor packages to lumino
* jupyterlab
* webpack and loaders

Plus regenerated yarn.lock from scratch.

### CODE REVIEW

**Comment by jasongrout** [comment]
It's hard to test this right now since the latest jlab prerelease is broken. Hopefully that is fixed soon.

**Comment by jasongrout** [comment]
Tested with the widget list in the classic notebook, which seems to work great!

File: packages/jupyterlab-manager/src/manager.ts [hunk 1]
```diff
@@ -113,16 +113,20 @@ class WidgetManager extends ManagerBase<Widget> implements IDisposable {
...
       this._handleKernelStatusChange(args);
     });
+    context.sessionContext.connectionStatusChanged.connect((sender, args) => {
+      this._handleKernelConnectionStatusChange(args);
+    });
+
 
-    if (context.session.kernel) {
-      this._handleKernelChanged({oldValue: null, newValue: context.session.kernel});
+    if (context.sessionContext.session?.kernel) {
```
Comment (SylvainCorlay) [inline packages/jupyterlab-manager/src/manager.ts]: Cool, I did not know this syntax.

Comment (jasongrout) [reply packages/jupyterlab-manager/src/manager.ts]: New in TS 3.7: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#optional-chaining

**Comment by jasongrout** [comment]
@meeseeksdev backport to 7.x

**Pull request merged**